badge_server/test_badges.py

```diff
@@ -595,3 +666,31 @@ def test_github_py2py3_py3_incompatible_fresh_nodeps(self):
         self.assertEqual(json_response['right_text'], 'self incompatible')
         self.assertEqual(json_response['right_color'], '#E05D44')
         self.assertLinkUrl(package_name, json_response['whole_link'])
+
+
+class TestDependencyOutdatedOrObsolete(BadgeImageDependencyIssueTestCase):
```

Comment:
I actually have an open PR that will add testing on Outdated/Obsolete dependencies, so can you a `# TODO: merge with future dependency test classes once PRs are merged`?

Reply:
Just saw your pending PR, just use yours is fine.